packages/hint-meta-theme-color/src/hint.ts

```diff
@@ -43,7 +44,10 @@ export default class MetaThemeColorHint implements IHint {
             const { resource } = event;
 
             if (!firstThemeColorMetaElement) {
-                context.report(resource, getMessage('metaElementNotSpecified', context.language));
+                context.report(
```

Comment:
We want this to be `off` if no manifest was found. I think we need at extra check for a `<link rel="manifest">` element. If we find one, then this should report a warning. If not, then nothing should be reported.

Reply:
I do the same in the [`apple-touch-icons` PR](https://github.com/webhintio/hint/pull/3301/files#diff-365998dbb143c5cd97435ad5567621e1R246)